Curtains/Curtains.Domain/Models/BaseEntity.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Domain.Models
+{
+    public class BaseEntity
+    {
+        public int Id { get; set; }
```

Comment:
what about title & description properties?

Reply:
But we don't have title & description properties in all models